state/imagestorage/image.go

```diff
@@ -42,27 +43,29 @@ var _ Storage = (*imageStorage)(nil)
 // database in the "imagemetadata" collection.
```

Comment:
Can you document the purpose of dbPrefix please?

Since the tests also pass through the empty string, do you think it is worthwhile having a package constant that has a meaningful name for "" ?  Perhaps 
```go
const NoPrefix = ""
```